### PR TITLE
Geodata on hover

### PR DESCRIPTION
**Pull request by VasavanThiru**
Change the GeoData class in order to use on_hover.
You can also see how to use it in this notebooks example: GeoData_on_hover.ipynb

geodata_on_hover

### CODE REVIEW

File: ipyleaflet/leaflet.py [hunk 1]
```diff
@@ -514,7 +514,7 @@ class GeoData(GeoJSON):
     geo_dataframe = Instance('geopandas.GeoDataFrame')
 
     def __init__(self, **kwargs):
-        super(GeoJSON, self).__init__(**kwargs)
+        super(GeoData, self).__init__(**kwargs)
```
Comment (martinRenou) [inline ipyleaflet/leaflet.py]: Good catch!! Is it why the on_hover was not working?

Comment (VasavanThiru) [reply ipyleaflet/leaflet.py]: yes definitely

Comment (VasavanThiru) [reply ipyleaflet/leaflet.py]: Now it works ! It deserves a tweet !  ;)

File: examples/GeoData_on_hover.ipynb [hunk 2]
```diff
@@ -0,0 +1,124 @@
+{
+ "cells": [
+  {
+   "cell_type": "code",
+   "execution_count": 1,
```
Comment (martinRenou) [inline examples/GeoData_on_hover.ipynb]: Do you mind removing the Notebook outputs?

Comment (VasavanThiru) [reply examples/GeoData_on_hover.ipynb]: Sure, it's clean now!

**Comment by janbrus** [comment]
I installed ipywidgets using conda some weeks ago, and still got problem showing the hoover data running the example jypyter notebook GeoData_on_hover.ipynb. Is it possible that this error has come back?

**Comment by davidbrochart** [comment]
I just tried, the notebook seems to work fine. Did you execute it entirely? The last cell enables the `on_hover`.

**Comment by janbrus** [comment]
When I now replaced the leaflet.py i got from Conda-forge with the one here on Github it is working OK,

**Comment by davidbrochart** [comment]
Then I suspect you have an installation issue. Could you try with a fresh conda environment?

**Comment by janbrus** [comment]
I tried now and installed ipywidget in a new environment with the same result, but I am not an expert on this. To me it looks like leaflet.py July 29th size 70565 is the one one conda-forge that causes trouble. leaflet.py from October 19th size 78229 works ok.

**Comment by martinRenou** [comment]
> When I now replaced the leaflet.py i got from Conda-forge with the one here on Github it is working OK,

Copying files around is the recipe to a broken installation, you should never do that :S 

What do you have when running `conda list | grep ipyleaflet` in the terminal?

**Comment by janbrus** [comment]
ipyleaflet                0.13.3             pyh9f0ad1d_0    conda-forge

**Pull request merged**